components/ParsePOSPayment/reducer.tsx

```diff
@@ -0,0 +1,84 @@
+import router from "next/router"
```

Comment:
I think this file does not belong under PasePOSPayment. It could be in the merchant page folder where it is being used.

Reply:
Noted. fixed.